src/serac/physics/benchmarks/CMakeLists.txt

```diff
@@ -6,14 +6,26 @@
 
 set(benchmark_depends serac_physics)
```

Comment:
Can you prefix all the variables here? So ``benchmark_depends`` to ``physics_benchmark_depends``.

Reply:
i renamed the files as well so the foreach loop still works.